airbyte-cdk/python/airbyte_cdk/models/airbyte_protocol.py

```diff
@@ -98,6 +99,28 @@ class Config:
     failure_type: Optional[FailureType] = Field(None, description="The type of error")
 
 
+class Type1(Enum):
```

Comment:
is there a way to avoid calling this `Type1` and instead call it like `EstimateType` or even `Type_`?

Reply:
It was auto-generated... I have no idea! Any suggestions that won't get overridden next time?

Reply:
can we call it `type_` in the JsonSchema? 👀

Reply:
Solved it with `title` in JSON schema!

Reply:
Solved it with `title` in JSON schema!

Reply:
Solved it with `title` in JSON schema!

Reply:
^ thanks, airplane internet :(